pages/api/transactions/[address].ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse, NextApiRequest } from 'next/types'
+import { getAddress } from 'services/bchdService'
+import { RESPONSE_MESSAGES } from 'constants/index'
+
+const { ADDRESS_NOT_PROVIDED_400 } = RESPONSE_MESSAGES
+
+export default async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  try {
+    const { address } = req.query
+    if (address === '' || address === undefined) {
+      res.status(ADDRESS_NOT_PROVIDED_400.statusCode).send({ message: ADDRESS_NOT_PROVIDED_400 })
+    }
+
+    const response = await getAddress(address)
```

Comment:
Values that come from `req.query` can be either `string` or `string[]`, so you have to address this second case or typescript will complain about it.

Reply:
I don't understand this, indeed here the tests are working, but the same thing is being done in #124 and the tests there are failing (I left a comment there)